napari/_vispy/vispy_camera.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+from vispy.scene import ArcballCamera, PanZoomCamera
+
+from .quaternion import quaternion2euler
+
+
+class VispyCamera:
+    """Vipsy camera for both 2D and 3D rendering.
+    """
+
+    def __init__(self, view, camera, dims):
```

Comment:
@sofroniewn could you add a docstring here? What is `view`? What is `camera`?

Reply:
done